documentation/http_client_metrics.markdown

```diff
@@ -0,0 +1,120 @@
+---
+layout: default
+title: "Puppet Server: HTTP Client Metrics"
+canonical: "/puppetserver/latest/http_client_metrics.html"
+---
+
+HTTP client metrics available in Puppet Server 5 allows users to measure how long it takes for Puppet Server to make requests to and receive responses from other services, such as PuppetDB.
+
+## Determining metrics IDs
+
+All of these metrics are of the form `puppetlabs.<SERVER ID>.http-client.experimental.with-metric-id.<METRIC ID>.full-response`.
+
+> **Note:** The `<METRIC ID>` describes what the metric measures. A metric ID is represented in the [status endpoint](./status-api/v1/services.markdown) as an array of strings, and in the metric itself the strings are joined together with periods. For instance, the metric ID of `[puppetdb resource search]` is `puppetdb.resource.search`, so the full metric name would be `puppetlabs.<server-id>.http-client.experimental.with-metric-id.puppetdb.resource.search.full-response`.
+
+During the course of handling a Puppet agent run, Puppet Server makes several calls to other services to get or store information.
```

Comment:
It might be worth calling out that the PuppetDB requests mentioned below would only apply if Puppet Server were configured as a storeconfigs and reports backend in the [puppet.conf](https://docs.puppet.com/puppetdb/latest/connect_puppet_master.html#edit-puppetconf) file.